package.json

```diff
@@ -32,7 +32,15 @@
     "node": ">=0.8.2"
   },
   "scripts": {
+    "start": "node ./tiddlywiki.js ./editions/tw5.com-server --listen",
     "dev": "node ./tiddlywiki.js ./editions/tw5.com-server --listen",
+    "pre": "node ./tiddlywiki.js ./editions/prerelease --listen",
```

Comment:
All the new commands are variants of the "start" command, so I think it might make sense to use that as a prefix: `start-prerelease`, `start-empty`, `start-tour` etc.

Having said that, there are a lot of them, which begs the question why we wouldn't introduce a single `npm run listen <edition>` command?

Reply:
I'll change it soon